jest.config.js

```diff
@@ -11,9 +11,8 @@ module.exports = {
   testEnvironment: 'jsdom',
   setupFilesAfterEnv: ['<rootDir>/jest-setup.ts'],
   moduleNameMapper: {
-    'react-i18next': '<rootDir>/__mocks__/react-i18next.ts',
-    ...pathsToModuleNameMapper(compilerOptions.paths, {
-      prefix: '<rootDir>',
+    ...pathsToModuleNameMapper(compilerOptions.path, {
```

Comment:
```suggestion
    ...pathsToModuleNameMapper(compilerOptions.paths, {
```
I had to do this for jest to run, with just `.path` it complains about "undefined" and stuff 🤷

Reply:
heh thanks